packages/react-divider/Spec.md

```diff
@@ -0,0 +1,203 @@
+# Divider Spec
+
+## Background
+
+The divider control is intended to replace the Separator control currently exported. Its purpose is to visually separate content.
+The purpose of redoing the component is to add minor structural updates and new engineering practices and update the naming conventions in the API to match naming conventions indicated by OpenUI research.
+
+## Prior Art
+
+[Github Epic Convergence Issue #16254](https://github.com/microsoft/fluentui/issues/16254)
+
+[Github Convergence Issue #15759](https://github.com/microsoft/fluentui/issues/15759)
+
+[Fluent UI Separator](https://developer.microsoft.com/en-us/fluentui#/controls/web/separator)
+
+[Open UI Component Matrix](https://open-ui.org/analysis/component-matrix)
+
+[Ant Design Divider](https://ant.design/components/divider/)
+
+[Fast Divider](https://explore.fast.design/components/fast-divider)
+
+[Semantic UI Divider](https://semantic-ui.com/elements/divider.html)
+
+## Comparison of v8 and v0
+
+- v8 - [Separator](https://developer.microsoft.com/en-us/fluentui#/controls/web/separator)
+- v0 - [Divider](https://fluentsite.z22.web.core.windows.net/0.52.1/components/divider/definition)
+
+## Sample Code
+
+Basic Examples:
+
+```html
+<Divider />
+<Divider vertical />
+<Divider>This is a divider with content</Divider>
+```
+
+## Variants
+
+The divider will have two main variants, rendering horizontally or vertically. The control is the same with the optional property `vertical` defining how it renders.
+
+Standard default horizontal
+
+```html
+<Divider />
+```
+
+Vertical
+
+```html
+<Divider vertical />
+```
+
+Color
+
+```html
+<Divider color="black" />
+```
+
+Content
+
+```html
+<Divider>My Content</Divider>
+```
+
+Icon
+
+```html
+<Divider><Icon /></Divider>
+```
+
+Important
+
+```html
+<Divider important>This is important!</Divider>
+```
+
+Inset
+
+```html
+<Divider inset />
+```
+
+Appearance
+
+```html
+<Divider apperance="subtle" />
+<Divider apperance="brand" />
+<Divider apperance="strong" />
+```
+
+## API
+
+From [Divider.types.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react-divider/src/components/Divider/Divider.types.ts)
+
+### Slots
+
+- root
+
+### Props
+
+```ts
+export interface DividerProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
+  /**
+   * Determines the alignment of the content within the divider.
+   * @defaultvalue 'center'
+   */
+  alignContent?: 'start' | 'end' | 'center';
+
+  appearance?: 'default' | 'subtle' | 'brand' | 'strong';
```

Comment:
Add a comment for `appearance`.

Also, for any prop that has a default value, add a `@defaultvalue` annotation to the comment.

Reply:
Updated